src/secrets/actions/thunks.ts

```diff
@@ -83,44 +83,45 @@ export const upsertSecret = (secretKey: string, secretValue: string) => async (
     const resp = await apiUpdateSecret({
       orgID: org.id,
       data: {
-        [secretKey]: secretValue,
+        [newSecret.key]: newSecret.value,
       },
     })
 
     if (resp.status !== 204) {
       throw new Error()
     }
 
+    // Secret values shouldn't be kept around in the store! -JF
+    const strippedSecret = {id: newSecret.key, key: newSecret.key}
```

Comment:
what's the point of setting an id and key to the same value?

Reply:
After some thought, there really wasn't a need for both id and key.  I'm pushing up a refactor to get rid of key and just rely on id instead.